prometheus/datadog_checks/prometheus/data/conf.yaml.example

```diff
@@ -1,89 +1,117 @@
 init_config:
-  # Leave empty
 
 instances:
-  # The prometheus endpoint to query
-  #
-  # - prometheus_url: http://service/prometheus
 
-  #   The namespace to be appended before all metrics namespace
+  ## @param prometheus_url - string - required
+  ## The prometheus endpoint to query.
   #
-  #   namespace: "service"
+  - prometheus_endpoint: http://localhost:10055/metrics
 
-  #  List of metrics to be fetched from the prometheus endpoint, if there's a value it'll be renamed
-  #  This list should contain at least one metric
+  ## @param namespace - string - optional
```

Comment:
for generic prometheus checks, the namespace is required: https://github.com/DataDog/integrations-core/blob/c5b88b02c902e0a499c2b4c8c166ec463cc3df37/datadog_checks_base/datadog_checks/base/checks/prometheus/base_check.py#L122-L125